### ISSUE TITLE
Separate notification category for service info

### ISSUE THREAD

**Issue by DoganM95** [binwiederhier/ntfy#931]
:bulb: **Idea**
Currently when the service is running, it creates a persisting "Listening for incoming notifications" notification, which is good. However this notification cannot be customized, nor turned off, as it seems to not be part of any android notification category, so it always shows up in the notifications bar and cannot be minimized/silenced.

Here is an attempt to turn off that notification, you can see even after turning off all categories, the persisting service notification is still there, until general ntfy is turned off:
![Screenshot_2023-10-25-15-06-19-247](https://github.com/binwiederhier/ntfy/assets/38842553/605fbbe5-745a-4905-82f1-237f7e8caeea)
![Screenshot_2023-10-25-15-06-25-831](https://github.com/binwiederhier/ntfy/assets/38842553/cc78fb7c-f087-4052-9968-e4bae7d170ac)
![Screenshot_2023-10-25-15-06-29-426](https://github.com/binwiederhier/ntfy/assets/38842553/3536e7c2-d167-440a-91ab-ee4e9cf16955)

:computer: **Target components**
Android app

**Comment by wunter8** [comment]
There is supposed to be a "Subscription Service" category that you can minimize or disable

![Screenshot_20231025-191541](https://github.com/binwiederhier/ntfy/assets/8421688/903b6e9b-8436-4131-b604-0f0485cc71b3)

What OS are you using?

**Comment by DoganM95** [comment]
Android 13 (On Nubia Red Magic 8 Pro)

**Comment by binwiederhier** [comment]
@DoganM95 Do you not see the subscription service?

**Comment by DoganM95** [comment]
@binwiederhier nope, no subscription service category for me. Just checked if this is some general issue on my phone but no, Tasker for example has lots of "System" and "User" categorized notifications that are customizable. Seems like your app does something different that does not work in my case.

**Comment by lewisleedev** [comment]
Unrelated but for those who are wondering why there is no notification categories at all: For some reason, Samsung decided to disable(hide) notifications category by default. You can enable it by enabling `Notifications - Advanced settings - Manage notification categories for each app`.

Hope this helps someone.
